apps/ex_wire/lib/ex_wire/adapter/tcp/server.ex

```diff
@@ -27,6 +27,20 @@ defmodule ExWire.Adapter.TCP.Server do
     {:ok, new_state}
   end
 
+  def init(state = %{is_outbound: false}) do
+    new_state =
+      state
+      |> listen_via_tcp()
```

Comment:
Can we ignore pipeline operator with single method?

Reply:
Updated in 8e5f416df3e4c59c0632fbb56d42a3babcb605d1